### DISCUSSION TITLE
If I rename my account and repo will it still keep all the stars and other data?

### DISCUSSION THREAD

**Discussion by CaravelKit** [community/community#22339] [New to GitHub]
<p>Hello,</p>
<p>I created a repo several months ago to verify the idea. As I saw interest and some amount of stars I decided to make it more serious. As a consequence, I rebranded the whole product (the codebase is still the same, I just changed the name). The url of the repo consists of account name and repo name. If I change both of them will the renamed repo keep all kudos it earned?</p>
<p>Thanks!</p>

**Comment by CaravelKit** [comment, marked as answer]
<p>For those who are interested in this question.</p>
<p>I already renamed account and repo, both. All the stats are the same, so no any problems.</p>

**Comment by mpboom** [comment]
<p>Hi <a class="mention" href="/caravelkit">@caravelkit</a> glad your issue got resolved and thanks for sharing the solution with the community!</p>
<p>Just for completeness, here is the guide: <a href="https://help.github.com/articles/renaming-a-repository/" rel="nofollow noopener">https://help.github.com/articles/renaming-a-repository/</a></p>
